tidy3d/components/geometry/primitives.py

```diff
@@ -270,7 +270,7 @@ def _points_unit_circle(
         self, num_pts_circumference: int = _N_PTS_CYLINDER_POLYSLAB
     ) -> np.ndarray:
         """Set of x and y points for the unit circle when discretizing cylinder as a polyslab."""
-        angles = np.linspace(0, 1, num_pts_circumference + 1)[:-1]
+        angles = 2 * np.pi * np.linspace(0, 1, num_pts_circumference + 1)[:-1]
```

Comment:
:smile:

Maybe `angles = np.linspace(0, 2 * np.pi, num_pts_circumference, endpoint=False)` for clarity? No `+ 1)[:-1]` needed.

Reply:
wow, took me too long to find out about `endpoint=False`.. thanks.